### PR TITLE
COMMERCE-1842 Make sure that shipment pull the correct order statuses and workflow approved

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
should be fixed in upstream

File: commerce-api/src/main/java/com/liferay/commerce/constants/CommerceShipmentConstants.java [hunk 1]
```diff
@@ -20,9 +20,12 @@
...
 
 	public static final int[] ALLOWED_ORDER_STATUSES = {
-		CommerceOrderConstants.ORDER_STATUS_TO_TRANSMIT,
+		CommerceOrderConstants.ORDER_STATUS_TRANSMITTED,
+		CommerceOrderConstants.ORDER_STATUS_AWAITING_FULFILLMENT,
 		CommerceOrderConstants.ORDER_STATUS_AWAITING_SHIPMENT,
-		CommerceOrderConstants.ORDER_STATUS_PARTIALLY_SHIPPED
+		CommerceOrderConstants.ORDER_STATUS_PARTIALLY_REFUNDED,
+		CommerceOrderConstants.ORDER_STATUS_PARTIALLY_SHIPPED,
+		CommerceOrderConstants.ORDER_STATUS_DISPUTED
```
Comment (ethanbustad) [inline commerce-api/src/main/java/com/liferay/commerce/constants/CommerceShipmentConstants.java:28]: @alecsloan these constants should be sorted alphabetically within the array (unless there's a good reason they are the way they are atm).

Comment (alecsloan) [reply commerce-api/src/main/java/com/liferay/commerce/constants/CommerceShipmentConstants.java:28]: no good reason just inserted based on original structure based on the unique term after ORDER_STATUS_ and appended new ones on the bottom. Fixed in #1376 also re-ordered the constants in SHIPMENT_STATUSES

**Pull request merged**